src/main/java/frc/robot/utils/NullableRotation2d.java

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.utils;
+
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.util.struct.StructSerializable;
+
+/** Wrapper over WPILib's Rotation2d to allow serialization of nulls. */
+public class NullableRotation2d implements StructSerializable {
+  final boolean isNull;
+  final Rotation2d value;
+
+  public NullableRotation2d(Rotation2d value) {
+    isNull = value == null ? true : false;
```

Comment:
When you find yourself saying: `someCondition ? true : false` you can just use `someCondition`.

So, `isNull = value == null`.